### ISSUE TITLE
Today’s Date is No Longer Emphasized on Date Picker

### ISSUE THREAD

**Issue by neildaniels** [k-next/kirby#4124]
## Description

The new date picker no longer highlights the current date on the calendar view. For example, I would expect that Feb 1 be highlighted, because it’s Feb 1 today.

Note that this is _not_ related to the selected date. Our editors just find it very helpful to know where _today_ is on the calendar view. The previous versions of Kirby would have a different day color showing this.

**Screenshots**  

![C2DDBF18-CFB6-4BAC-858B-A696F187E202](https://user-images.githubusercontent.com/417182/152041308-f5039c5e-3030-4b14-a84e-c65f528b48f0.jpeg)

## Your setup
**Kirby Version**  
`3.6.2`

**Console output**  

**Your system (please complete the following information)**
 - Device: Mac
 - OS: macOS
 - Browser: Safari

**Additional context**

**Comment by bastianallgeier** [comment]
✅

**Comment by sebastiangreger** [comment]
I'm just testing the 3.6.3-rc.1 and would like to suggest an improvement to the "current day highlight" re accessibility. At the moment, the current date is only highlighted by a very subtle colour highlight (light yellow vs. white; contrast between those two foreground colours is only 2.77:1) and an almost inconceivable increase in boldness.

Quick-fire idea: maybe add an underline?

**Comment by neildaniels** [comment]
Kirby 3.6.3 seems to have landed with _only_ the underline effect. I think it helps to have both the color change _and_ underline. Not sure why the color was removed.

I added this to my custom Panel CSS file to restore a color effect:

```css
.k-calendar-day[aria-current=date] .k-button {
    color: var(--color-yellow-400);
}
```

<img width="257" alt="Screen Shot 2022-03-22 at 10 15 07 AM" src="https://user-images.githubusercontent.com/417182/159538695-958ffd7b-8176-4d1f-af6d-946bae98da2b.png">
